.github/workflows/ci.yml

```diff
@@ -16,17 +16,17 @@ jobs:
 
   test:
     needs: [lint]
-    runs-on: ${{ matrix.os }}
+    runs-on: ubuntu-latest
```

Comment:
why you changed this dinamic OS to fixed ubuntu ?

Reply:
Because the `supercharge/mongodb-github-action@1.3.0` on github Actions do not work with MacOS or Windows